tests/init.c

```diff
@@ -434,7 +434,7 @@ main (int argc, char **argv)
     {
       int ret;
 
-      ret = printf ("%d:%d", getuid(), getgid());
+      ret = printf ("%d:%d", getuid (), getgid ());
```

Comment:
Wonder how this eluded the clang-format CI job?

Reply:
Since this is merged https://github.com/containers/crun/pull/938 i think this change can be removed once rebased.